app/AndroidManifest.xml

```diff
@@ -438,7 +462,8 @@
 
         <receiver
             android:name="org.commcare.provider.RefreshToLatestBuildReceiver"
-            android:exported="false">
+            android:permission="org.commcare.dalvik.permission.UPDATE_TO_LATEST_BUILD"
```

Comment:
`RefreshToLatestBuildReceiver` is a internal dev only tool and introducing a permission here will break the underlying scripts. Given it was marked as `exported=false` earlier, is there a good reason change that to `true` now ?

Reply:
I might not have enough context here, but my understanding is that this receiver is for external use, its documentation also states that it should respond to requests via `adb`, in that case it needs to be exported.

> Trigger from command line with:
>  *   adb shell am broadcast -a org.commcare.dalvik.api.action.RefreshToLatestBuildAction

Considering that this is for dev use only, what's the usage? To make sure the feature is working as expected?

Reply:
It's a adb command to update the app to latest without interacting with UI and was meant to increase dev efficiency - https://github.com/dimagi/commcare-android/blob/master/scripts/ccc#L63

Reply:
Deleting this receiver then, thanks